tests/ssl/generate_tls_keys.sh

```diff
@@ -109,13 +109,11 @@ gen_crl() {
         -out "${OUT_DIR}/$signer/crl.pem"
 }
 
-declare -a names=(mysql postgresql acra-writer acra-writer-2 
-    acra-writer-revoked acra-server ocsp-responder intermediate-ca vault 
-    acra-client redis)
+declare -a names=(consul)
```

Comment:
I think it was changed for local generation?

Reply:
Yes, you are right. Thanks for noticing!